client/editor.lua

```diff
@@ -1,6 +1,6 @@
 RegisterCommand("record", function()
     StartRecording(1)
-    TriggerEvent('QBCore:Notify', "Started Recording!", "success")
+    TriggerEvent('QBCore:Notify', Lang:t('cruise.started'), "success")
```

Comment:
These translations should be `Lang:t('editor.started')` rather than `Lang:t('cruise.started')`

Applies to all 4 translations in this file.